IS0401Test.py

```diff
@@ -70,7 +70,7 @@ def _registry_mdns_info(self, port, priority=0):
         info = ServiceInfo(service_type,
                            "NMOSTestSuite{}.{}".format(port, service_type),
                            socket.inet_aton(default_ip), port, 0, 0,
-                           txt, "nmos-test.local.")

```

Comment:
I'm not sure about this. Using anything other than '.local' for mDNS seems prone to incompatibility and confusion. I'd have preferred to see a SAN for the .local name if we're going to allow testing via mDNS with hostnames.

Reply:
> For Apache at least I'd typically expect a separate chain file (via SSLCertificateChainFile).

According to https://httpd.apache.org/docs/current/mod/mod_ssl.html#sslcertificatefile:
> The files may also include intermediate CA certificates, sorted from leaf to root. This is supported with version 2.4.8 and later, and obsoletes SSLCertificateChainFile.

Reply:
> I'm not sure about this. Using anything other than '.local' for mDNS seems prone to incompatibility and confusion. I'd have preferred to see a SAN for the .local name if we're going to allow testing via mDNS with hostnames.

You're right. I was half-way between testing mDNS and uDNS. Does adding the SAN for a .local name make sense?

Reply:
I believe it would be fine in theory. The challenge is whether every unicast DNS resolver knows to send that request to 224.0.0.251.

Reply:
OK. I've tried putting the `ServiceInfo.server` back to `"nmos-test.local."` and adding `DNS.2:nmos-test.local` to the mocks' certs' SAN, and that works for me. How do you think I should proceed actually?

I've also double checked that the mocks work when the chain is appended to their certs but not if they're not.

Reply:
If the `.local` SAN can be used successfully then I'd suggest including that approach in the repo

Reply:
This needs passing through to the `openssl.cnf` file then... and should only be done for the mocks certs, not the api certs? How best to allow that to be configured via the `SAN` environment variable and command-line args to `generateCerts`?